bower.json

```diff
@@ -9,5 +9,8 @@
     "showdown": "1.5.1",
     "sinonjs": "1.17.1",
     "clockpicker-seconds": "~0.1.6"
+  },
+  "resolutions": {
+    "route-recognizer": "^0.2.3"
```

Comment:
idk, I did a bower install and it asked me to resolve a version conflict.  I did a `!` to put it in the `bower.json`